lib/mix/tasks/compile.elchemy.ex

```diff
@@ -32,7 +32,6 @@ defmodule Mix.Tasks.Compile.Elchemy do
     Mix.Task.run "deps.compile"
     IO.puts "-- Elchemy compilation complete --\n"
 
-    if version == "~> 1.7", do: IO.warn "Elixir 1.7 version doesn't work with Elchemy. You have to use lower."
-
+    if Regex.match?(~r/1.[7-9].[0-9]+$/, version), do: IO.warn "Elixir 1.7 version doesn't work with Elchemy. You have to use lower."
```

Comment:
I'd prefer it to be more specific. "Doesn't work" is not particularly descriptive.

I think something like "Elchemy's functionality is limited in Elixit 1.7. To support the full experience please use different Elixir version."

Also, we don't have to match higher than 1.7. Elixir 1.8 is promised to work fine as well